src/client/components/pages/entities/relationships.js

```diff
@@ -33,8 +33,10 @@ function EntityRelationships({contextEntity, relationships}) {
 					>
 						<Relationship
 							link
+							attributes={relationship.attributeSetId !== null ? relationship.attributeSet.relationshipAttributes : null}
 							contextEntity={contextEntity}
 							relationshipType={relationship.type}
+							showAttributes={relationship.attributeSetId !== null}
```

Comment:
Let's make it a bit more lax, for example accept `undefined` as well:
(We don't expect an attributeSetId would ever be the number `0`, which would evaluate to false)

```suggestion
							showAttributes={Boolean(relationship.attributeSetId)}
```